client/modules/food/reducers/packing.js

```diff
@@ -142,6 +159,21 @@ export default (state = {}, action) => {
         saving: false,
         saveError: null,
       }
+    case RECEIVED_REQUEST: {
+      return {
+        ...state,
+      }
+    }
+    case RECEIVED_SUCCESS:
+      return {
+        ...state,
+        ids: without(state.ids, action.response.packages._id).concat(action.response.packages),
```

Comment:
The ids don't change, so all you'd need to do here is clear saving and error states